besu/src/test/java/org/hyperledger/besu/cli/PublicKeySubCommandTest.java

```diff
@@ -52,10 +52,10 @@
           + System.lineSeparator()
           + "This command outputs the node public key. Default output is standard output."
           + System.lineSeparator()
-          + "      --to=<FILE>   File to write public key to instead of standard output"
-          + System.lineSeparator()
           + "  -h, --help        Show this help message and exit."
           + System.lineSeparator()
+          + "      --to=<FILE>   File to write public key to instead of standard output"
+          + System.lineSeparator()
```

Comment:
what is the actual difference here?

Reply:
basically the --to goes below help now because of ordering of the new picoCli

Reply:
fair enough